pkg/accounting/accounting.go

```diff
@@ -365,7 +365,8 @@ func (c *creditAction) Apply() error {
 	c.accounting.metrics.CreditEventsCount.Inc()
 
 	if c.price.Cmp(c.accountingPeer.reservedBalance) > 0 {
-		c.accounting.logger.Error(ErrOverRelease, "for peer %v")
+		c.accounting.logger.Debug(ErrOverRelease.Error(), "peer", c.peer, "error", ErrOverRelease)
```

Comment:
Accounting calls happen way to often. Adding duplicate debug and error logs here is not right. The logs would go out of control. API package has different requirements.

Remove all the debug logs here. You can add the extra information in the error logs itself.

Reply:
> Remove all the debug logs here. You can add the extra information in the error logs itself.

Given that, then any err can go to the first argument of the .Error log instead of being an additional key/value pair?

Reply:
Yes, error should go into the first argument. If any additional key-value pairs need to be added we can always add them.